README.md

```diff
@@ -77,22 +89,132 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
+6. Create another folder for the output folders and files
+7. Run the docker container's `odm_workflow` app specifying  either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries, and the two named volumes
+8. Copy the resulting files off the output named volume to the local folder
+9. Clean up the named volumes
+
+#### For example: <a name="opendm_can_shp_example" />
+
+You can download a sample dataset of files (archived) with names corresponding to those listed here from CyVerse using the following command.
+Be sure to replace **<username>** and **<password>** with your CyVerse username and password.
+```bash
+curl -X GET -u '<username>:<password>' https://data.cyverse.org/dav/iplant/projects/aes/cct/diag/sample-data/scif_odm_test_data.tar.gz > scif_odm_test_data.tar.gz
+gunzip scif_odm_test_data.tar.gz
+tar -xf scif_odm_test_data.tar
+```
+
+In this example we're going to assume that we're using a shapefile named `plot_shapes.shp`, and that we have our drone images in a folder named `/IMG`.
+
+We will need one other file for this example, the `experiment.yaml` file containing some additional information.
+Copy the following content into the experiment.yaml file:
+```text
+%YAML 1.1
+---
+pipeline:
+    studyName: 'S7_20181011'
+    season: 'S7_20181011'
+    germplasmName: Sorghum bicolor
+    collectingSite: Maricopa
+    observationTimeStamp: '2018-10-11T13:01:02-08:00'
+```
+
+Step 1 requires creating two named Docker volumes to use when processing.
+If you already have one or more empty named volumes you can skip this step.
+```bash
+docker volume create my_input
+docker volume create my_output
+``` 
+
+Step 2 involves copying the drone images into a folder:
+```bash
+mkdir -p /inputs
+cp /IMG/* /inputs/
+```
+
+Step 3 copies the optional shapefile files into the same folder as the drone images:
+```bash
+cp /plot_shapes.* /inputs/
+``` 
+
+In step 4 we copy the experiment.yaml file into the same folder as the drone images:
+```bash
+cp experiment.yaml /inputs/
+``` 
+
+In step 5 we copy the source files onto the input named volume:
+```bash
+docker run --rm -v /inputs:/sources -v my_input:/input --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp /sources/* /input/'
+``` 
+
+In step 6 we create a local folder to hold the output from processing:
+```bash
+mkdir -p /output
+```
+
+In step 7 we run the workflow to generate the orothomosaic image using ODM (OrthoDroneMap) and calculate plot-level canopy cover:
+```bash
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /inputs:/scif/data/odm_workflow/images -v my_output:/output -e INPUT_VOLUME=my_input -e OUTPUT_VOLUME=my_output -e "INPUT_IMAGE_FOLDER=/images" -e "OUTPUT_FOLDER=/output" agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output
+```
+and we wait until it's finished.
```

Comment:
Wish list: A command to monitor the progress.

Reply:
Added issue: https://github.com/AgPipeline/issues-and-projects/issues/243